coldfront/core/resource/migrations/0004_auto_20230524_0136.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 3.2.5 on 2023-05-24 01:36
```

Comment:
Give human-readable names to these migrations? I _think_ they were caused by the upgrade to the history module?